### PR TITLE
Add inscriptis

### PR DESCRIPTION
**Pull request by saippuakauppias**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -102,6 +104,7 @@ or external resources:
   at https://github.com/scrapinghub/article-extraction-benchmark/pull/4
 - go-readability: https://github.com/go-shiori/go-readability
 - Readability.js: https://github.com/mozilla/readability
+- inscriptis: https://github.com/weblyzard/inscriptis
```
Comment (lopuhin) [inline README.rst]: @saippuakauppias for both this and html2text, maybe it makes sense to put them into the same category with ``html-text``, as they are not intended as article extraction libraries, more like html to text libraries - so I'd either add a similar remark that they "extracts the full text of HTML page", or even put in a separate list below this one (results table is good as-is 👍 ).

Comment (saippuakauppias) [reply README.rst]: Done :)

Comment (saippuakauppias) [reply README.rst]: `html2text` fix it in own PR after the merge of this one :)

**Review by lopuhin — APPROVED** [review]
Great, thanks @saippuakauppias 👍

**Pull request merged**